include/picongpu/particles/ParticlesFunctors.hpp

```diff
@@ -126,7 +126,7 @@ struct LogMemoryStatisticsForSpecies
         const std::shared_ptr<T_DeviceHeap>& deviceHeap
     ) const
     {
-#if( PMACC_CUDA_ENABLED == 1 )
+#if( BOOST_LANG_CUDA || BOOST_COMP_HIP)
```

Comment:
Maybe we could introduce smth like  `PMACC_CUDA_OR_HIP_ENABLED` to express that `BOOST_LANG_CUDA || BOOST_COMP_HIP` ? This expression is repeating in several placed

Reply:
Discussed offline: this is not needed.